database/classes/Course.js

```diff
@@ -0,0 +1,88 @@
+const faker = require('faker');
+
+const genLangs = (smallGen = false) => {
+  const languages = [
+    'Arabic',
+    'French',
+    'Portuguese (European)',
+    'Chinese (Simplified)',
+    'Italian',
+    'Vietnamese',
+    'German',
+    'Russian',
+    'Hebrew',
+    'Spanish',
+    'Hindi',
+    'Japanese',
+    'Turkish',
+    'Gujarati',
+    'Polish',
+    'Persian',
+    'Kannada',
+    'Romanian',
+  ];
+  var usedLangs = [];
+  if (smallGen) {
+    usedLangs.push(languages[Math.floor(Math.random() * (languages.length))]);
+  } else {
+    languages.forEach((lang) => {
+        Math.random() < .05 ? usedLangs.push(lang) : null;
+    });
+  }
+  return usedLangs;
+}
+const genRecentViews = () => {
+  return Math.floor(Math.random() * 100000000) + 60000;
+};
+const genDescription = (smallGen = false) => {
+  return smallGen ? faker.lorem.paragraphs(1) : faker.lorem.paragraphs(4);
```

Comment:
I like the use of a boolean to determine whether you are creating larger / smaller records

Reply:
Thank you! I wanted to be able to make the first chunk a little bigger just so that there were pages that were a bit more visually interesting.